src/plugins/add-cal/add-calendar-fr.hbs

```diff
@@ -0,0 +1,74 @@
+---
+{
+	"title": "Ajout au calendrier",
+	"language": "fr",
+	"category": "Plugiciels",
+	"description": "",
+	"tag": "add-cal",
+	"parentdir": "add-cal",
+	"altLangPrefix": "add-cal",
+	"dateModified": "2021-04-28"
+}
+---
+
+<h2>Événement incluant un endroit détaillé</h2>
+<p>L'exemple suivant présente un événement incluant un emplacement détaillé (nom, adresse, région, province et code postale).</p>
+<div class="wb-addcal provisional mrgn-bttm-md" typeof="Event">
+	<h3 property="name">Canux20</h3>
+	<p property="description">CanUX (prononcé «canucks», comme l&apos; équipe de hockey) est un événement indépendant, local, à but non lucratif et dirigé par des bénévoles, créé pour les personnes qui souhaitent apprendre et partager leur expertise sur l'expérience utilisateur et la façon dont notre profession contribue à faire les choses fonctionnent mieux pour tout le monde.</p>
+	<p property="location" typeof="Place">
+		<span property="name">Centre Shaw</span><br>
+		<span class="address" property="address" typeof="PostalAddress">
+			<span property="streetAddress">55, promenade du Colonel-By</span><br>
+			<span property="addressLocality">Ottawa</span>,
+			<span property="addressRegion">ON</span><br>
+			<span property="postalCode">K1N 9J2</span>
+		</span>
+	</p>
+	<dl>
+		<dt>Début</dt>
+		<dl><time property="startDate" datetime="2020-01-27 07:00Z">Le lundi 27 janvier 2020, 7:00am (HNE)</time></dl>
+		<dt>Fin</dt>
+		<dl><time property="endDate" datetime="2020-01-29 12:00Z">Le mercredi 29 janvier 2020, 12:00pm (HNE)</time></dl>
+	</dl>
+</div>
+
+<h2>Example d'événement incluant seulement un lieu</h2>
+<p>L'exemple suivant présente un événement incluant le nom d'un lieu sans adresse complète.</p>
+<div class="wb-addcal provisional mrgn-bttm-md" typeof="Event">
+	<h3 property="name">Igloo Baratanga</h3>
+	<p property="description">Une activité de percussions pour toute la famille! Participez à un cercle de tambours et réchauffez-vous. Aucune expérience musicale requise et sourires garantis! Viens jouer avec nous!</p>
+	<p><strong>Quand</strong>&nbsp;: <time property="startDate" datetime="2020-02-08 13:00Z">Le dimanche 8 février 2021 | de 13:00 (HNE)</time> à <time property="endDate" datetime="2020-02-08 13:30Z">13:30 (HNE)</time></p>
+	<P property="location"><strong>Où</strong>&nbsp;: Le Domaine des flocons au parc Jacques-Cartier (Gatineau)</p>
```

Comment:
Suggestion, usually we use lowercase for tag name

Reply:
Changes have been made and submitted via commit amend.